src/main/resources/hudson/scm/SubversionSCM/ModuleLocation/help-stickyDepth.html

```diff
@@ -0,0 +1,5 @@
+<div>
+  Choose if <tt>--set-depth</tt> (sticky) or <tt>--depth</tt> (non-sticky) option should be used for switch and update commands. Default value is sticky.
+  More information can be found
```

Comment:
@vvv444 Please, use `<code>` instead of  `<tt>`. I'll try to merge this PR soon.